crates/bins/wick/src/commands/install.rs

```diff
@@ -0,0 +1,151 @@
+use anyhow::Result;
+use clap::Args;
+use structured_output::StructuredOutput;
+use wick_config::WickConfiguration;
+
+use crate::options::reconcile_fetch_options;
+
+#[derive(Debug, Clone, Args)]
+#[clap(rename_all = "kebab-case")]
+#[group(skip)]
+pub(crate) struct Options {
+  #[clap(flatten)]
+  pub(crate) oci: crate::oci::Options,
+
+  /// Path or OCI url to manifest or wasm file.
+  #[clap(action)]
+  path: String,
+}
+
+#[allow(clippy::unused_async)]
+pub(crate) async fn handle(
+  opts: Options,
+  settings: wick_settings::Settings,
+  span: tracing::Span,
+) -> Result<StructuredOutput> {
+  let xdg = wick_xdg::Settings::new();
+  let bin_dir = xdg.global().root().join("bin");
+  std::fs::create_dir_all(&bin_dir)?;
+
+  span.in_scope(|| info!(app = opts.path, to = %bin_dir.display(), "installing wick app"));
+
+  let oci_opts = reconcile_fetch_options(&opts.path, &settings, opts.oci, false, None);
+  let package = crate::oci::pull(opts.path, oci_opts).await?;
+  let path = package.path();
+  let config = WickConfiguration::fetch(path.to_string_lossy(), Default::default())
+    .await?
+    .into_inner();
+
+  let config = match config {
+    WickConfiguration::App(config) => config,
+    _ => anyhow::bail!("{} is not an application configuration", path.display()),
+  };
+
+  let bin_path = bin_dir.join(config.name());
+
+  #[cfg(not(target_os = "windows"))]
+  {
+    let sh = format!(
+      r#"
+    #!/bin/sh
+
+    wick run {}
+    "#,
+      path.display()
```

Comment:
should this be the registry path if that exists? This would allow for running a `latest` and keep that automatically updated.

Reply:
No, i don't think so. No app should auto-update itself by default once installed on a local filesystem. It would be a bad user experience to suddenly change or break an application without giving the user a chance to prevent it.